### DISCUSSION TITLE
nonan.io - Data Exploration web app

### DISCUSSION THREAD

**Discussion by pietz** [f5/unovis#229] [Show and tell]
Hey team,

I just pushed a product I've been working on in open alpha. It's a data exploration tool built in SvelteKit and it uses Unovis for charts! Just wanted to say thanks for providing such a great library. I tried several others before and Unovis made me the happiest overall. Especially those super smooth transitions ;)

https://www.nonan.io

Best,
Pietz

**Comment by rokotyan** [comment]
Thanks for sharing that @pietz! We're happy to hear that you chose Unovis over other libraries. The look and feel of the components is indeed one of our top priorities. 

Keep up the excellent work in nonan.io, and thanks for helping us make Unovis better by sharing your suggestions and filing bug reports!

P.S. Do you mind if we share your feedback on social media or use it as a testimonial on our website in the future?
